features/FEATURE_BLE/ble/Gap.h

```diff
@@ -321,6 +269,617 @@ class GapAdvertisingData;
  * controller or by the peer.
  */
 class Gap {
+
+public:
+
+    /**
+     * Definition of the general handler of Gap related events.
+     */
+    struct EventHandler {
+        /** Called when a scanning device request a scan response.
+         *
+         * @param event Scan request event: @see ScanRequestEvent_t for details.
+         */
+        virtual void onScanRequest(const ble::ScanRequestEvent_t& event) { }
```

Comment:
Would this method (and similar) need to be renamed to something like `onScanRequestReceived` to be consistent with other names like `onPeriodicAdvertisingSyncEstablished`?

Reply:
@donatieng Fixed.